bundles/Billing/Service/BillingService.php

```diff
@@ -815,10 +816,12 @@ public function getProjectWorklogsWithMetadata($projectId)
         foreach ($worklogs as $worklog) {
             $issue = $worklog->issue;
 
-            foreach ($epics as $epic) {
-                if ($epic->key === $issue->epicKey) {
-                    $issue->epicName = $epic->fields->{$epicNameCustomFieldId};
-                    break;
+            if (!empty($issue->epicKey)) {
```

Comment:
A comment about what's going on here would be nice, e.g. “Enrich issues with epic name”.